### DISCUSSION TITLE
remember shrink mode across different connections

### DISCUSSION THREAD

**Discussion by nikkdevic** [rustdesk/rustdesk#1979] [General]
Default view should be 'Shrink' for new connections because it shows full screen of the remote system.
Currently default is 'Original'.
When accessing a larger screen only part of the screen is shown.

Or it should remember the selection for the next new connection.

**Comment by rustdesk** [comment]
not remembered? which build?

**Comment by nikkdevic** [comment]
I installed rust just 2 days ago.
I mean remember the 'shrink' option selected from previous remote access for connecting to a new remote system, when I entered new system number and connected it had default as 'original'.
But since systems have varying display sizes 'Shrink' should be default.

**Comment by rustdesk** [comment]
@team  check out tv/anydesk behavior first before taking action, or do nothing change.

**Comment by nikkdevic** [comment]
Thanks for looking into this and thanks for the time and effort you guys spent into making this 👍
Most of the times I'm using my laptop to connect to desktop systems which have a larger screen.
This happens only the first time I connect to a new system.
When I connect to the new system I cant see the Taskbar of the new system untill i set 'Shrink' view.
So I think Shrink should be default.(anydesk does it)

**Comment by nikkdevic** [comment]
Untill Settings menu can be added can you set the default in software as 'shrink' so the full display is shown. Anydesk default is 'Shrink'. It can be changed by users as needed.
![AnydeskSettings](https://user-images.githubusercontent.com/97282126/200154047-99922180-246a-41d4-ae19-50b0c20e60a8.png)

**Comment by yashpatel1599** [comment]
@nikkdevic @rustdesk is possible to make default mode to shrink not to be original ?
